webapp/platform/types/src/admin.ts

```diff
@@ -114,11 +114,18 @@ export type IndexedPluginAnalyticsRow = {
     [key: string]: PluginAnalyticsRow;
 }
 
+export enum AnalyticsVisualizationType {
+    Count = 'count',
+    LineChart = 'line_chart',
+    DoughnutChart = 'doughnut_chart',
+}
+
 export type PluginAnalyticsRow = {
     id: string;
     name: React.ReactNode;
-    icon: string;
-    value: number;
+    icon?: string;
+    value: any;
+    visualizationType: AnalyticsVisualizationType;
```

Comment:
Being purist, aren't all these optional? I mean... old plugins will not send them, right?

Reply:
Fair. I tried to update the type to match the "new" version. So `icon` becomes optional and the `visualizationType` is required. Everything else is untouched, apart from `value` now being `any`. Even that could be strictly typed but it would require us to couple it with `chart.js` `ChartData` type. Not sure if it's worth it.

But you are right that old plugins will likely not respect this. Not sure if making everything optional is any better, though; maybe a little safer.

Reply:
Better to just tell it like it is: you have no idea what you're going to get, so you need to handle it all. Making the `visualizationType` optional seems the right way to go. As for `value`? I see the point of not wanting to tie it to chart.js, but on the other hand we're going to have plugins sending a specific version of `ChartData`, right? We might want to purposefully make that a breaking change if some person updates ChartData on the server and breaks an older plugin...

Reply:
> We might want to purposefully make that a breaking change if some person updates ChartData on the server and breaks an older plugin...

@cpoile To do that, we'd need to pass the whole expected data format in a test on this side, right? Otherwise, nothing will break until a real plugin does.

Reply:
I can do that, I'm already passing a bunch of stuff in the test case added here. In general I feel e2e tests are our best insurance. They saved us a bunch of times.

Reply:
I am keeping the type as `any` because our `LineChart` component accepts that so the change here would be significantly more involved.